src/App.vue

```diff
@@ -1,9 +1,41 @@
 <template>
-  <div class="m-auto my-6 flex max-w-screen-md flex-col content-center justify-center">
-    <TodoHeader />
-  </div>
+  <main class="mx-auto my-6 max-w-screen-sm">
+    <div class="flex flex-col content-center justify-center gap-12 px-4">
+      <TodoHeader @add-todo="addTodo" />
+      <TodoListEmpty v-if="isTodoListEmpty" />
+      <TodoListItem
+        v-for="item in todoList"
+        :key="item.id"
+        :title="item.title"
+        :description="item.description"
+        :priority="item.priority"
+        :checked="item.checked"
+      />
+    </div>
+  </main>
 </template>
 
 <script setup lang="ts">
+import { computed, ref } from 'vue'
 import TodoHeader from '@/components/TodoHeader.vue'
+import TodoListEmpty from '@/components/TodoListEmpty.vue'
+import TodoListItem from '@/components/TodoListItem.vue'
+import type Todo from '@/types/Todo'
+
+const todoList = ref<Todo[]>([])
+const isTodoListEmpty = computed(() => !todoList.value.length)
+
+// Function that simulates a todo add for testing
+function addTodo() {
+  let priorities = ['High', 'Medium', 'Low']
```

Comment:
If a variable doesn't change it's value, always use `const` please

Reply:
And also you can move it outside the function in order not to declare it every time when you call this function